core-kotlin-modules/core-kotlin-10/src/test/kotlin/com/baeldung/obtainTheOSName/ObtainTheOSNameUnitTest.kt

```diff
@@ -0,0 +1,45 @@
+package com.baeldung.obtainTheOSName
+
+import org.apache.commons.lang3.SystemUtils
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import org.slf4j.LoggerFactory
+
+class ObtainTheOSNameUnitTest {
```

Comment:
We follow these guidelines for class names:
https://google.github.io/styleguide/javaguide.html#s5.3-camel-case

(camel case even for abbreviations)

```suggestion
class ObtainTheOsNameUnitTest {
```